### PR TITLE
Add localization instructions

### PR DESCRIPTION
**Pull request by bucanero**

### CODE REVIEW

File: LOCALIZATION.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+
+**Developer**
+
+1. `concepts`
+2. `develop`
+3. `tools`
+4. `api`
+5. `tutorials`
+6. `videos`
```
Comment (Dispa1r) [inline LOCALIZATION.md:55]: The chapters should be :
* 1.concepts
* 2.develop
* 3.tutorials
* 4.tools
* 5.api
* 6.intergrator

File: LOCALIZATION.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+
+**Community**
```
Comment (Dispa1r) [inline LOCALIZATION.md:58]: And it seems that community chapter has been removed
```suggestion

```

File: LOCALIZATION.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+1. `community`
+
+
+**Exchanges & Integration**
```
Comment (Dispa1r) [inline LOCALIZATION.md:63]: These chapters are same as before

**Review by Dispa1r — CHANGES_REQUESTED** [review]

File: LOCALIZATION.md (hunk 1 above)
Comment (bucanero) [reply LOCALIZATION.md]: Fixed 👍

File: LOCALIZATION.md (hunk 2 above)
Comment (bucanero) [reply LOCALIZATION.md]: fixed

File: LOCALIZATION.md (hunk 3 above)
Comment (bucanero) [reply LOCALIZATION.md:58]: 👍

**Pull request merged**